KomaMRIBase/src/datatypes/Sequence.jl

```diff
@@ -83,7 +83,7 @@ Sequence() = Sequence([Grad(0, 0)])
 """
     str = show(io::IO, s::Sequence)
```

Comment:
This function does not return a string.

Reply:
Addresed in [b1f8e54](https://github.com/cncastillo/KomaMRI.jl/pull/297/commits/b1f8e547e00a31d226fd93af7200a9e648dac2b7)